src/lib.rs

```diff
@@ -274,6 +274,33 @@ impl Element {
         self.children.iter_mut().find(predicate)
     }
 
+    /// Transverse element using an xpath-like string: root/child/a
+    pub fn find(&self, path: &str) -> Option<&Element> {
+        Self::find_path(&path.split('/').collect::<Vec<&str>>(), &self)
+    }
+
+    pub fn find_text(&self, path: &str) -> &str {
```

Comment:
`error_chain` is great, we could make great use of `links` here and for `xml-rs`'s errors.